src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -95,6 +95,11 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
     return this._defaultCountry;
   }
 
+  /**
+   * Specifies the countries which will be available for selection. Accepts an array of
```

Comment:
How about?

Specifies the countries available for selection. If not specified, the list includes all ISO country codes.

Reply:
We lose the portion about the array needing to be of the country codes with that. Are we ok with that since the input name has `ISO` in it? 

Also, we say "the list" - I know what you mean but do we think consumers will?

Reply:
Good point; it's probably worth calling out again just to avoid any potential confusion.

Specifies an array of ISO country codes. If not specified, the selector includes all ISO country codes.

hopefully a little better?